src/Servers/Kestrel/Core/src/Internal/Http3/Http3Connection.cs

```diff
@@ -81,9 +89,6 @@ internal long HighestStreamId
         {
             try
             {
-                // Ensure TimeoutControl._lastTimestamp is initialized before anything that could set timeouts runs.
```

Comment:
I don't think is as important for Http3Connection (we call ProcessStreamsAsync right after the constructor).